SU-SOE/stanford_soe_helper
# SOE-2651: rows 22-37

# NOT READY FOR REVIEW

# Summary
- This adds rows 22-37 to SASS

# Needed By (Date)
- When possible

# Criticality
- Fixed on prod, need to put into code.

# Steps to Test
- Check out this branch ( 7.x-2.x-tweaks-03-22-x ) onto an SoE site
- visit [rows 22-37](https://docs.google.com/spreadsheets/d/1O7VSJD25zNH722Fbg4XcHl3M6KkAZAGKtzRCkU9M45c/edit#gid=0) and verify changes

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2651

## Related PRs

## More Information

## Folks to notify
@kerri-augenstein


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
